lib/liquid_voting/voting.ex

```diff
@@ -71,17 +71,33 @@ defmodule LiquidVoting.Voting do
 
   ## Examples
 
-      iex> list_votes("https://docs.google.com/document/d/someid", "a6158b19-6bf6-4457-9d13-ef8b141611b4")
+      iex> list_votes_by_proposal("https://docs.google.com/document/d/someid", "a6158b19-6bf6-4457-9d13-ef8b141611b4")
       [%Vote{}, ...]
 
   """
```

Comment:
I renamed 'list_votes' to 'list_votes_by_proposal' to differentiate it from the new 'list_votes_by_participant/2'.

Reply:
👍